ManatiCR/artavia
# AR-55 impacto internacional #2

[AR-55](https://manati.atlassian.net/browse/AR-55)
[Figma designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
  This PR depends of this one: [AR-73](https://github.com/ManatiCR/artavia/pull/6)
---
**What was done**
- Create the contet type caso de estudio
- Create the list of caso de estudio, imapacto internacional page.
- Create a custom block to use as a hero image
- Page to test: http://artavia.docker/es/impacto-internacional and the display node of publicaciones

**Steps to test**
- Run `sudo rm -r web && composer update && ./scripts/artavia_local_install.sh`
- Create caso de estudio nodes, should have the fields you can see in the design and the field industria
- In the page http://artavia.docker/es/impacto-internacional you should see first a block with sample text where should be an image will be added as a background for that block with styles.
- Should be able to filter by industria, only 5 nodes should be displayed per page.
